scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Nice summary. Minor suggestions below:</li><li>Would rephrase sentence to: In particular, we will use <a href="https://github.com/ifzhang/FairMOT" target="_blank">FairMOT</a> which is a state-of-the-art tracking model with high accuracy and inference speed.</li><li>Then delete the "which detects objects and learns their re-ID features"</li><li>Remove sentence "We also show how to save and load the trained model for inference on a second video.  "</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>